app/app.go

```diff
@@ -367,7 +367,7 @@ func New(
 	app.PaymentKeeper = *paymentmodulekeeper.NewKeeper(
 		appCodec,
 	)
-	paymentmodule := paymentmodule.NewAppModule(appCodec, app.PaymentKeeper)
+	paymentmod := paymentmodule.NewAppModule(appCodec, app.PaymentKeeper)
```

Comment:
this appears to be because we're overriding the import, which makes a lot of sense to change. Do we have extra CI here that we don't have on the main branch?

Reply:
it's better for readability I guess and I have extra linters on my IDE. I can revert if not needed.

Reply:
no we should change this upstream too, overriding the import is bad (harmless in this case, but bad).